### PR TITLE
`Feature` SRS (EXPOSUREAPP-13454)

### PR DESCRIPTION
**Pull request by mtwalli**
👷🏼‍♂️ 🚧 
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13454

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionDispatcherViewModel.kt [hunk 1]
```diff
@@ -40,6 +46,26 @@ class SubmissionDispatcherViewModel @AssistedInject constructor(
         )
     }
 
+    fun onSelfTestClicked() = launch {
+        try {
+            srsLocalChecker.check()
+            // proceed with SRS type  param
```
Comment (Ein-Tim) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionDispatcherViewModel.kt:52]: Double whitespace
```suggestion
            // proceed with SRS type param
```

**Review by Ein-Tim — COMMENTED** [review]
One comment.

File: Corona-Warn-App/src/main/res/values/srs_submission_strings.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    <!-- XTXT: Positive test no answer body -->
+    <string name="srs_positive_test_no_answer_body">Warnen Sie andere mit Ihrem eigenen positiven Schnell- oder PCR-Test, auch wenn Ihr Test nicht registriert war oder Ihr Testergebnis nicht in der App zugestellt wurde.</string>
+
+    <!-- Submission Consent Screen -->
+    <!-- XHED: Submission consent header -->
+    <string name="srs_submission_consent_header">"Ihr Einverständnis"</string>
+    <!-- XHED: Submission consent headline -->
+    <string name="srs_submission_consent_headline_text">Andere warnen</string>
+    <!-- XHED: Submission consent disclaimer sub headline -->
+    <string name="srs_submission_consent_disclaimer_subheadline_text">Bevor Sie andere warnen können, ist ihr Einverständnis erforderlich.</string>
```
Comment (MikeMcC399) [inline Corona-Warn-App/src/main/res/values/srs_submission_strings.xml:37]: "ihr Einverständnis" => "Ihr Einverständnis"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Typo - see in-line comment

File: Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    <!-- XTXT: Positive test no answer body -->
+    <string name="srs_positive_test_no_answer_body">Warnen Sie andere mit Ihrem eigenen positiven Schnell- oder PCR-Test, auch wenn Ihr Test nicht registriert war oder Ihr Testergebnis nicht in der App zugestellt wurde.</string>
+
+    <!-- Submission Consent Screen -->
+    <!-- XHED: Submission consent header -->
+    <string name="srs_submission_consent_header">"Ihr Einverständnis"</string>
+    <!-- XHED: Submission consent headline -->
+    <string name="srs_submission_consent_headline_text">Andere warnen</string>
+    <!-- XHED: Submission consent disclaimer sub headline -->
+    <string name="srs_submission_consent_disclaimer_subheadline_text">Bevor Sie andere warnen können, ist ihr Einverständnis erforderlich.</string>
```
Comment (MikeMcC399) [inline Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml:37]: "ihr Einverständnis"?

**Comment by mtwalli** [comment]
> Typo - see in-line comment

Fixed

File: Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    <!-- XTXT: Type selection button other text -->
+    <string name="srs_other_text">Sonstige / keine Angabe</string>
+    <!-- XTXT: Type selection next button text -->
+    <string name="type_selection_next_button_text">Weiter</string>
+
+    <!-- Srs Submission Cancel Dialog -->
+    <!-- XHED: Cancel submission dialog title -->
+    <string name="srs_cancel_dialog_title">Warn-Vorgang abbrechen?</string>
+    <!-- XTXT: Cancel submission dialog message -->
+    <string name="srs_cancel_dialog_message">Sind Sie sich sicher, dass Sie den Warn-Vorgang abbrechen wollen?\n\Wenn Ihr Test positiv war, können Sie mit einer Warnung helfen, Infektionsketten zu unterbrechen.</string>
```
Comment (MikeMcC399) [inline Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml:85]: Is `\n\Wenn` correct?
Previously it was `\n\nWenn` and it looks like one `\n` was intended to be deleted.

Same issue in file `values` directory as well.

Comment (mtwalli) [reply Corona-Warn-App/src/main/res/values-de/srs_submission_strings.xml:85]: https://github.com/corona-warn-app/cwa-app-android/pull/5709

**Pull request merged**